README.md

```
@@ -1,3 +1,84 @@
-# java-janggi
+# 장기 게임
+
+## 학습 목표
+
+**OOP - Object-Oriented Programming**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)